AirCasting/SDSync/Parsing/MiniSDCardMeasurementsParser.swift

```diff
@@ -68,14 +69,13 @@ class MiniSDCardMeasurementsParser: SDMeasurementsParser {
                                        time: measurementInfo[MiniSDCardCSVFileFactory.Header.time])
     }
     
-    func enumerateSessionLines(lines: [String], action: (String, String) -> Void) {
+    func enumerateSessionLines(lines: [String], action: (String?, String) -> Void) {
         var lastKnownUUID: String?
         lines.forEach { lineString in
             let measurementInfo = lineString.split(separator: ",")
             if measurementInfo.count == firstLineColumns {
                lastKnownUUID = getUUID(lineString: lineString)
             }
-            guard let lastKnownUUID else { return }
```

Comment:
This had to be removed because ABMini doesn't send UUID with each data pack so the data without it wasn't saved.

Reply:
Yeah, but that was the idea to keep the "lastKnownUUID" for those packs to be able to assign them to a session. What do you do with packs without uuids?